app/models/secure_data.rb

```diff
@@ -0,0 +1,28 @@
+# Note: model uses UUID for id which is generated by Postgres. Note that any changes to
+# the way the UUID is generated need to ensure that it generated by a secure random system.
+class SecureData < ApplicationRecord
+  def self.create_and_store(hash)
+    secure_data = new
+    secure_data.store(hash)
+    secure_data.save
+    secure_data.id
+  end
+
+  def self.for(id)
```

Comment:
I don't see this method used anywhere

Reply:
Thanks for that. I remember where I meant to use it - I've updated the `LegalAidApplication` model to make it a little cleaner.